package.json

```diff
@@ -1,62 +1,62 @@
-        "eslint-config-google": "^0.14.0",
-        "eslint-plugin-node": "^11.1.0",
-        "eslint-plugin-prettier": "^3.1.3",
-        "eslint-plugin-react": "^7.20.0",
-        "gts": "^2.0.2",
-        "ts-jest": "^26.1.0"
-    },
-    "scripts": {
-        "lint": "eslint 'src/**/*.{ts?(x),js?(x)}' && tsc --noEmit",
-        "lint-fix": "eslint 'src/**/*.{ts?(x),js?(x)}' --fix",
-        "test": "jest",
-        "verify": "yarn lint && yarn test",
-        "verify-dev": "eslint 'src/**/*.{ts?(x),js?(x)}'; tsc --noEmit; jest # run all checks no matter the results",
-        "build": "gatsby clean && gatsby build",
-        "clean": "gatsby clean",
-        "start": "gatsby build && gatsby serve",
-        "start-dev": "gatsby develop",
-        "start-safe": "yarn verify && yarn clean && yarn start # ensure all checks pass, and serve a clean build"
-    },
-    "dependencies": {
-        "gatsby": "^2.23.3",
-        "gatsby-plugin-client-side-redirect": "^1.1.0",
-        "gatsby-plugin-google-analytics": "^2.3.4",
-        "gatsby-plugin-react-helmet": "^3.3.4",
-        "gatsby-plugin-sass": "^2.3.4",
-        "gatsby-plugin-sitemap": "^2.4.5",
-        "gatsby-source-filesystem": "^2.3.11",
-        "gatsby-transformer-remark": "^2.8.15",
-        "gatsby-transformer-yaml": "^2.4.4",
-        "jest": "25.5.4",
-        "node-sass": "^4.14.1",
-        "query-string": "^6.13.1",
-        "react": "^16.13.1",
-        "react-dom": "^16.13.1",
-        "react-helmet": "^6.1.0",
-        "react-select": "^3.1.0",
-        "rehype-sanitize": "^3.0.1",
-        "rehype-stringify": "8.0.0",
-        "remark-breaks": "^1.0.5",
-        "remark-parse": "^8.0.2",
-        "remark-rehype": "^7.0.0",
-        "tslint": "^6.1.2",
-        "typescript": "^3.9.5",
-        "unified": "^9.0.0",
-        "uuid": "^8.1.0",
-        "webpack": "^4.43.0"
-    }
+  "engines": {
+    "node": "14.4.0"
+  },
+  "devDependencies": {
+    "@types/jest": "^26.0.0",
+    "@types/react": "^16.9.36",
+    "@types/react-dom": "^16.9.8",
+    "@types/react-helmet": "^6.0.0",
+    "@types/theme-ui": "^0.3.3",
+    "@types/uuid": "^8.0.0",
+    "@typescript-eslint/eslint-plugin": "^3.2.0",
+    "@typescript-eslint/parser": "^3.2.0",
+    "eslint": "^7.2.0",
+    "eslint-config-google": "^0.14.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^3.1.3",
+    "eslint-plugin-react": "^7.20.0",
+    "gts": "^2.0.2",
+    "ts-jest": "^26.1.0"
+  },
+  "scripts": {
+    "lint": "eslint 'src/**/*.{ts?(x),js?(x)}' && tsc --noEmit",
+    "lint-fix": "eslint 'src/**/*.{ts?(x),js?(x)}' --fix",
+    "test": "jest",
+    "verify": "yarn lint && yarn test",
+    "verify-dev": "eslint 'src/**/*.{ts?(x),js?(x)}'; tsc --noEmit; jest # run all checks no matter the results",
+    "build": "gatsby clean && gatsby build",
+    "clean": "gatsby clean",
+    "start": "gatsby build && gatsby serve",
+    "start-dev": "gatsby develop -H 0.0.0.0",
```

Comment:
this is the change, rest is code shifting

see https://stackoverflow.com/questions/52961824/run-gatsby-with-docker-compose